Gemfile

```diff
@@ -70,3 +70,5 @@ group :test do
   gem "selenium-webdriver"
   gem "webdrivers"
 end
+
+gem "sidekiq"
```

Comment:
We also need to start redis in order to start sidekiq.

refs.
redis is in-memory data structure store.
https://redis.io/